master/cache.lua

```diff
@@ -177,7 +179,7 @@ local function fetch_and_store_marathon_apps()
 
        local port = ports[portIdx]
        if not port then
-          ngx.log(ngx.NOTICE, "Cannot find port at port index '" .. portIdx .. "' for app '" .. appId .. "'")
+          ngx.log(ngx.NOTICE, "Cannot find port at port index '" .. (portIdx - 1) .. "' for app '" .. appId .. "'")
```

Comment:
Should we maybe add "Marathon" here  (`"Cannot find port at Marathon port index..."`) to make the intent of the log message unambiguously clear?